### PR TITLE
Update CryptoSwift to 1.8.0

### PR DESCRIPTION
**Pull request by 417-72KI**
Diff: [krzyzanowskim/CryptoSwift@1.7.2...1.8.0](https://github.com/krzyzanowskim/CryptoSwift/compare/1.7.2...1.8.0)

### CODE REVIEW

**Comment by SimplyDanny** [comment]
You also need to update [bazel/repos.bzl](https://github.com/realm/SwiftLint/blob/ed8ed010f1fb3429419375736472814281b8da61/bazel/repos.bzl#L36).

File: bazel/repos.bzl [hunk 1]
```diff
@@ -37,8 +37,8 @@ def swiftlint_repos(bzlmod = False):
         name = "com_github_krzyzanowskim_cryptoswift",
         sha256 = "974d7892d7b4307a4fbe20ac389c9f3ab851bb17cb08e7f55443adb616b3f6ec",
```
Comment (jpsim) [inline bazel/repos.bzl:38]: You'll have to update this value.

**Comment by 417-72KI** [comment]
> You also need to update [bazel/repos.bzl](https://github.com/realm/SwiftLint/blob/ed8ed010f1fb3429419375736472814281b8da61/bazel/repos.bzl#L36).

Thank you, and fixed!

File: bazel/repos.bzl (hunk 1 above)
Comment (417-72KI) [reply bazel/repos.bzl]: Fixed :bow:

**Review by SimplyDanny — APPROVED** [review]

**Pull request merged**